docs/book/src/user/quick-start.md

```diff
@@ -158,6 +158,23 @@ Test to ensure the version you installed is up-to-date:
 clusterctl version
 ```
 
+{{#/tab }}
+{{#tab windows}}
+
+#### Install clusterctl binary with curl on windows
```

Comment:
```suggestion
#### Install clusterctl binary with curl on Windows
```

nit

Reply:
> nit

@killianmuldoon was earlier committing with "Windows" only but saw everything else in the doc is small for eg : linux, macOS, homebrew.

Should I make changes and commit ?